src/util/compat.test.js

```diff
@@ -0,0 +1,169 @@
+// @flow
+
+import {toCompat, fromCompat} from "./compat";
+import type {Compatible} from "./compat";
+
+describe("compat utilities", () => {
+  type FooV1 = {foo: number};
+  type FooV2 = {bar: {foo: number}};
+  const type = "Foo Plugin's Experiment";
+  const v1 = "v1";
+  const v2 = "v2";
+  function fooHandler(x: FooV1): FooV2 {
```

Comment:
Can we call this `convertV1ToV2` or something similar?

Reply:
Done